test/e2e/Tests/Tests/LargeOutputOrchestratorTests.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
+
+[Collection(Constants.FunctionAppCollectionName)]
+public class LargeOutputOrcehstratorTests
+{
+    private readonly FunctionAppFixture _fixture;
+    private readonly ITestOutputHelper _output;
+
+    public LargeOutputOrcehstratorTests(FunctionAppFixture fixture, ITestOutputHelper testOutputHelper)
+    {
+        _fixture = fixture;
+        _fixture.TestLogs.UseTestLogger(testOutputHelper);
+        _output = testOutputHelper;
+    }
+
+    [Theory]
+    [InlineData(65)] // Provide a value slightly exceeding the 64 KB Azure Queue Storage limit to trigger use of blob storage instead at Azure Storage backend.
+    public async Task LargeOutputStatusQueryTests(int size)
+    {
+        using HttpResponseMessage response = await HttpHelpers.InvokeHttpTriggerWithBody("LargeOutputOrchestrator_HttpStart", size.ToString(), "application/json");
+
+        Assert.Equal(HttpStatusCode.Accepted, response.StatusCode);
+        string statusQueryGetUri = await DurableHelpers.ParseStatusQueryGetUriAsync(response);
+
+        await DurableHelpers.WaitForOrchestrationStateAsync(statusQueryGetUri, "Completed", 30);
+
+        string largeOutput = GenerateLargeString(size);
+
+        var orchestrationDetails = await DurableHelpers.GetRunningOrchestrationDetailsAsync(statusQueryGetUri);
+        
+        // Verify that large orchestrator outputs stored in blob storage are correctly returned via statusQueryGetUri
+        Assert.Contains(largeOutput, orchestrationDetails.Output);
+    }
+
+    [Theory]
+    [InlineData(4608)]// This value exceeds the default 4 MB, as the test sets the threshold to 6 MB.
+    [Trait("DTS", "Skip")] 
```

Comment:
Skip DTS here because it also has its own gRPC client, which defaults to a [1 MB payload limit](https://dev.azure.com/msazure/One/_git/AAPT-DTMB?path=/src/Backend/Microsoft.DTMB.BackendAPI/ServiceConfiguration.cs&version=GBmain&_a=contents&line=426&lineStyle=plain&lineEnd=427&lineStartColumn=1&lineEndColumn=1). To support larger payloads, we may need to update the DTS SDK so it can read from this setting and adjust its MaxPayloadSizeBytes accordingly. Is this correct? @cgillum

Reply:
DTS uses the default gRPC configuration, which limits to 4 MB payloads. It's the DTS backend APIs which validate that all payloads are <= 1 MB, and this is non-configurable for users, by design. Let's not worry about DTS for now. We will need more than just SDK changes to support larger payloads in DTS.